src/pages/TicketPage/TicketPage.tsx

```diff
@@ -276,6 +267,19 @@ export const TicketPage = () => {
                         </Grid>
                     </AccordionDetails>
                 </TicketAccordionWrapper>
```

Comment:
May change the color of view seat layout button to grey for uniformity.

Reply:
Done